app/components/post_body/system_message_helpers.js

```diff
@@ -144,11 +144,25 @@ const renderArchivedMessage = (postBodyProps, styles, intl) => {
     return renderMessage(postBodyProps, styles, intl, localeHolder, values);
 };
 
+const renderUnarchivedMessage = (postBodyProps, styles, intl) => {
```

Comment:
Please add tests to system_message_helpers.test.js

Reply:
done.